### ISSUE TITLE
ObjectNotfoundException in RemoveObjectAsync tooltip needs to be removed

### ISSUE THREAD

**Issue by dthemg** [minio/minio-dotnet#685]
When removing an object using `RemoveObejctAsync` no error is thrown
```
await _minioClient.RemoveObjectAsync(new RemoveObjectArgs().WithBucket("my-bucket").WithObject(Guid.NewGuid().ToString()));
```
It is however stated in the tooltip that an exception should be thrown if the object isn't found:

![image](https://user-images.githubusercontent.com/13008236/188171804-b0da9467-451d-4908-9429-be2f00e9c20c.png)

**Comment by ebozduman** [comment]
@dthemg 

This is a tooltip bug. `"ObjectNotFound"` exception does not belong there and needs to be removed.

So, the existing behavior is correct.

**Comment by ebozduman** [comment]
Fixed by #690
